src/BUILD

```diff
@@ -292,8 +292,13 @@ cc_binary(
 cc_binary(
     name = "libcustom_node_east_ocr.so",
     srcs = [
+        "custom_nodes/common/utils.hpp",
+        "custom_nodes/common/buffersqueue.hpp",
```

Comment:
Why do we require buffers queue & internal manager when we don't use them for now in EAST OCR custom node?

Reply:
Inside east_ocr.cpp file we include now `"../common/utils.hpp"`. In this utility header file we use CustomNodeLibraryInternalManager from `"customNodeLibraryInternalManager.cpp"`. This however requires to compile `buffersqueue.*` files because they are used in manager implementation.

Reply:
I would extract custom node library internal manager stuff out of utility.hpp so we do not have to include manager when not needed.